pyjs9/__init__.py

```diff
@@ -225,17 +231,17 @@ def send(self, obj, msg='msg'):
         >>> js9.send({'cmd': 'SetColormap', 'args': ['red']})
         'OK'
         """
-        if obj == None:
+        if obj is None:
             obj = {}
         obj['id'] = self.id
         jstr = json.dumps(obj)
         try:
-            url = urllib.urlopen(self.host + '/' + msg, jstr)
+            url = urlopen(self.host + '/' + msg, jstr.encode('utf8'))
```

Comment:
PS, in case you didn't know, the Python standard library HTTP packages are pretty bad and most projects (e.g. [astroquery](http://astroquery.readthedocs.org/en/latest/) use the third-party [requests package](http://docs.python-requests.org/en/latest/) instead. If you need to do testing from Python you should use `requests` to simplify your life ... for `pyjs9` I guess the HTTP requests are simple enough for now that it's not needed, but it also wouldn't be hard to use it as a dependency or copy some files in `pyjs9/extern` to have it bundled and avoid an extra dependency.